src/annotator/integrations/pdf.js

```diff
@@ -198,12 +200,22 @@ export class PDFIntegration extends TinyEmitter {
   }
 
   /**
-   * Return true if the text in a range lies within the text layer of a PDF.
+   * Trim `range` to remove leading or trailing empty content, then check to see
+   * if that trimmed Range lies within a single PDF page's text layer. If so,
+   * return the trimmed Range.
    *
    * @param {Range} range
    */
-  canAnnotate(range) {
-    return canDescribe(range);
+  getAnnotatableRange(range) {
+    try {
+      const trimmedRange = trimRange(TextRange.fromRange(range).toRange());
+      if (canDescribe(trimmedRange)) {
+        return trimmedRange;
+      }
+    } catch (err) {
+      // no-op
```

Comment:
What errors are we expecting here, only the `RangeError` if there is no non-whitespace content? I suggest we check for that and re-throw any unexpected errors.